src/main/java/seedu/address/model/issue/IssueList.java

```diff
@@ -88,11 +112,17 @@ public void setIssues(IssueList replacement) {
     /**
      * Replaces the contents of this list with {@code issues}.
      * {@code issues} must not contain duplicate issues.
+     *
```

Comment:
Missing punctuation and captialisation. Param description should be separated from param name.